### PR TITLE
Full nav bar styling for top and bottom halves

### PR DESCRIPTION
**Pull request by SeanMcCarthy3223**
- This includes a bug fix to correctly resize the module responsively.
- Major styling changes for the bottom half of the bar 
- Minor Style changes to the top half of the bar
-Tweaks to the function of the search bar (Added State)

### CODE REVIEW

File: client/components/CategoryList.jsx [hunk 1]
```diff
@@ -4,15 +4,15 @@ import Category from './Category.jsx';
...
-      <Category name="Cat 9"/>
+      <Category name="Wands"/>
+      <Category name="Quidditch"/>
+      <Category name="Spells"/>
+      <Category name="Accessories"/>
+      <Category name="Rarities"/>
+      <Category name="Food"/>
+      <Category name="Furniture"/>
+      <Category name="Lamps"/>
+      <Category name="Lord of The Rings"/>
```
Comment (jxkim) [inline client/components/CategoryList.jsx]: Mapping through an array of categories in state would make this look nicer. Your call!

File: public/nav-style.css [hunk 2]
```diff
@@ -126,38 +163,162 @@
...
+#nav-location-text {
+  display: flex;
+  flex-flow: column nowrap;
+}
+
+#nav-location-text-top {
+  margin: 0;
+  padding: 0;
+  margin-top: 5px;
+  font-size: 9pt;
```
Comment (jxkim) [inline public/nav-style.css]: Pt or px?

File: public/nav-style.css [hunk 3]
```diff
@@ -126,38 +163,162 @@
...
+  align-items: flex-end;
   width: 810px;
   border: 1px solid white;
   color: white;
+}
+
+#nav-language-select {
+  padding: 5px;
+  padding-bottom: 0;
+  padding-right: 8px;
```
Comment (jxkim) [inline public/nav-style.css]: Merge padding into 1 line

File: client/components/CategoryList.jsx (hunk 1 above)
Comment (SeanMcCarthy3223) [reply client/components/CategoryList.jsx]: yes it is planned!

File: public/nav-style.css (hunk 2 above)
Comment (SeanMcCarthy3223) [reply public/nav-style.css]: pt is for fonts

File: public/nav-style.css (hunk 3 above)
Comment (SeanMcCarthy3223) [reply public/nav-style.css]: Style Police? ok will do!

**Pull request merged**